File: src/main/java/net/liyze/basin/core/scan/RestartCommand.java
package net.liyze.basin.core.scan;

import com.itranswarp.summer.annotation.Component;
import net.liyze.basin.core.Basin;
import net.liyze.basin.core.Command;
import org.jetbrains.annotations.NotNull;

import java.util.List;

@Component
public class RestartCommand implements Command {
    @Override
    public void run(@NotNull List<String> args) {
        Basin.restart();
    }

    @Override
    public @NotNull String Name() {
        return "restart";
    }
}

File: src/main/kotlin/net/liyze/basin/core/scan/RestartCommand.kt
package net.liyze.basin.core.scan

import net.liyze.basin.core.Command
import net.liyze.basin.core.restart


class RestartCommand : Command {
    override fun run(args: List<String?>) {
        restart()
    }

    override fun Name(): String {
        return "restart"
    }
}
